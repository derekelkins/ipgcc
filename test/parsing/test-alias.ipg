S -> A A@1 A@2 { value = A@1.x };
A -> { x = . };

%postamble_begin
abc
