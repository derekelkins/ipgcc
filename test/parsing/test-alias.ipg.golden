S
  -> A@0[0, EOI]
     A@1[A@0.END, EOI]
     A@2[A@1.END, EOI]
     { value = A@1.x };

A
  -> { x = .[0] };